### DISCUSSION TITLE
Newsletter field breaks form submission

### DISCUSSION THREAD

**Discussion by jmaas-onoffice** [onOffice-Web-Org/oo-wp-plugin#390] [_Discarded pitches]
As described in #386, the newsletter field prevents forms from being sent.

Since [research is needed](https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/386#issuecomment-1340675421) to understand why this is happening, I propose to spend at most a week trying to fix this issue.

If the issue cannot be resolved in a week, please report what you learned and what you think would be necessary to understand and fix the problem.

**Comment by hungnc89** [comment]
@jmaas-onoffice 
About issue #375, we found problem like you know as could not submit form when have newsletter field active on UI and we tried to fix it right then, when we did issue #375. 

And after that, I see that it was mentioned by you in issue #386, and because requirement of issue #375 is just only to delete the newsletter field on the admin form interface so I ignored it and didn't try to fix it anymore, and think we'll do it at #386.

It is a pretty serious bug that needs more time to figure out.

I will revert the changed of 375
